manifests/x/Xiaocai/MiniCADSee/2025R1.2/Xiaocai.MiniCADSee.installer.yaml

```
@@ -14,9 +14,9 @@ ProductCode: CAD迷你看图
 Installers:
 - Architecture: x86
   InstallerUrl: https://clientdown.appol.com/a_0509/MiniCADSee.exe
-  InstallerSha256: CAB67AC7052BB780CB57FC7CF053BAA0069C2CB2C40AE1DC0054BE175662100F
+  InstallerSha256: A65396E4BEFA57DA7234BDEB04834859CF77C21B42DDD5095EE91775A0A55F94

```

```suggestion
 InstallerSha256: FFC801007E279A1F474B37976CF8BC0E5F668B66603DE74052FAD5865304A347
```

(Automated response - build 979.)